web/cobrands/buckinghamshire/assets.js

```diff
@@ -43,7 +43,43 @@ fixmystreet.assets.add($.extend(true, {}, defaults, {
     asset_item: 'grit bin'
 }));
 
+var pin_prefix = fixmystreet.pin_prefix || document.getElementById('js-map-data').getAttribute('data-pin_prefix');
+var llpg_stylemap = new OpenLayers.StyleMap({
+    'default': new OpenLayers.Style({
+        fillColor: "#FFFF00",
+        fillOpacity: 0.6,
+        strokeColor: "#000000",
+        strokeOpacity: 0.8,
+        strokeWidth: 2,
+        pointRadius: 6
+    }),
+    'select': new OpenLayers.Style({
+        externalGraphic: pin_prefix + "pin-spot.png",
+        fillColor: "#55BB00",
+        graphicWidth: 48,
+        graphicHeight: 64,
+        graphicXOffset: -24,
+        graphicYOffset: -56,
+        backgroundGraphic: pin_prefix + "pin-shadow.png",
+        backgroundWidth: 60,
+        backgroundHeight: 30,
+        backgroundXOffset: -7,
+        backgroundYOffset: -22,
+        popupYOffset: -40,
+        graphicOpacity: 1.0,
+
+        label: "${Feature_id}",
+        labelOutlineColor: "white",
+        labelOutlineWidth: 2,
+        labelYOffset: -15,
+        fontSize: '11px',
```

Comment:
I used this in the screenshot:
```suggestion
        labelOutlineWidth: 3,
        labelYOffset: 65,
        fontSize: '15px',
```